examples/gno.land/p/gov/executor/context.gno

```diff
@@ -0,0 +1,81 @@
+package executor
+
+import (
+	"errors"
+	"std"
+
+	"gno.land/p/demo/context"
+)
+
+type propContextKey string
```

Comment:
By the way, I need to rewrite the p/demo/context because it is currently unsafe to use. We are not storing the private context key type, which means we could potentially steal a key from another realm. 

I have a solution that involves using PrevRealm() as the prefix for a key. 

There is nothing to change in this PR, but we must remember that it is currently unsafe.